apps/cnspec/cmd/scan.go

```diff
@@ -63,6 +63,7 @@ func init() {
 	scanCmd.Flags().MarkHidden("category")
 	scanCmd.Flags().Int("score-threshold", 0, "If any score falls below the threshold, exit 1.")
 	scanCmd.Flags().Bool("share", false, "create a web-based private reports when cnspec is unauthenticated. Defaults to false.")
+	scanCmd.Flags().String("output-target", "", "Set output target to which the asset report will be sent. Currently only supports AWS SQS topic URLs and local files")
```

Comment:
Until we support more native os paths file `--output-file report.json` we should mark this flag as hidden. Its not common to require a `file://` prefix for a local file.